src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -89,6 +92,15 @@ class WorkspaceMembersPage extends React.Component {
         this.getWorkspaceMembers();
     }
 
+    /**
+     * On Search change state of searchValue
+     *
+     * @param {String} searchValue
+     */
+    onSearch(searchValue = '') {
```

Comment:
as per the style guide methods should be named for what they do e.g. `updateSearchValue()` and not what they handle or the user action taken.

Reply:
Done @marcaaron